jenkins/ci.suse.de/cloud-crowbar9.yaml

```diff
@@ -54,13 +54,12 @@
     cloudsource: stagingcloud8
     upgrade_cloudsource: stagingcloud9
     cloud_env: cloud-crowbar-ci-slot
-    scenario_name: std-lmm
+    scenario_name: standard
     controllers: '1'
     computes: '1'
     ses_enabled: false
-    enabled_services: database,rabbitmq,keystone,swift,monasca,glance,cinder,neutron,nova,horizon
-    extra_params: |
-      ssl_enabled=false
```

Comment:
How is removal of this relevant to monasca removal?

Reply:
As ssl working already I decided enable it also. Can create separate PR if needed.

Reply:
No need for separate PR, but mentioning it in commit message would be nice